src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -90,8 +90,6 @@ void IProgressStepRunnerWrapper.ChangeHost(IProgressControlHost host)
         #region IHost
         public event EventHandler ActiveSectionChanged;
 
-        public Dispatcher UIDispatcher { get; }
```

Comment:
Miscellaneous: should have been removed in a previous PR, when I removed the property from the `IHost` interface.